### PR TITLE
feat: Modify ID verification endpoint to allow access to the current user.

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [CX-2239](https://artsyproduct.atlassian.net/browse/CX-2239)

### CODE REVIEW

File: src/schema/v2/me/__tests__/sendIdentityVerficationEmailMutation.test.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+} = {}): string => {
+  const mutation = gql`
+    mutation {
+      sendIdentityVerificationEmail(
+        input: {
+          userID: ${JSON.stringify(userID)}
+        }
+      ) {
+        confirmationOrError{
+							... on IdentityVerificationEmailMutationSuccessType{
```
Comment (MounirDhahri) [inline src/schema/v2/me/__tests__/sendIdentityVerficationEmailMutation.test.ts:31]: **thought:** is this formatted weirdly only for me 🤔

**Review by MounirDhahri — APPROVED** [review]
Thanks Sam 🙌

File: src/schema/v2/me/__tests__/sendIdentityVerficationEmailMutation.test.ts (hunk 1 above)
Comment (rajsam003) [reply src/schema/v2/me/__tests__/sendIdentityVerficationEmailMutation.test.ts:31]: It looks weird on Github only. 🧐

File: src/schema/v2/me/sendIdentityVerficationEmailMutation.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+const IdentityVerificationEmailType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "IdentityVerificationEmail",
+  fields: () => ({
+    ...InternalIDFields,
+    state: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Where the identity verification is in its lifecycle",
```
Comment (olerichter00) [inline src/schema/v2/me/sendIdentityVerficationEmailMutation.ts:24]:
```suggestion
      description: "Identity verification lifecycle state",
```

**Review by olerichter00 — APPROVED** [review]
Looking good 🌟 

I've just added a suggestion for a description.

**Pull request merged**